token/cli/src/config.rs

```diff
@@ -57,6 +57,8 @@ impl<'a> Config<'a> {
                 eprintln!("error: Could not find config file `{}`", config_file);
                 exit(1);
             })
+        } else if let Some(config_file) = &*solana_cli_config::CONFIG_FILE {
```

Comment:
Any particular reason you did this instead of the original `if let Some(ref config_file) = *solana_cli_config::CONFIG_FILE`?

Reply:
oh, no i just hadnt seen that form anywhere else. i can change it back if you prefer

Reply:
Naw, I don’t feel strongly, just curious.